src/fec_base.h

```diff
@@ -796,14 +796,14 @@ void FecCode<T>::decode_prepare(
     const vec::Vector<T>& fragments_ids = context.get_fragments_id();
     for (int i = 0; i < this->n_data; ++i) {
         const int j = fragments_ids.get(i);
-        auto data = props[j].get(ValueLocation(offset, j));
+        auto data = props[j].get(offset);
 
-        // Check if the symbol is a special case whick is marked by "@".
+        // Check if the symbol is a special case whick is marked by 1, i.e. true
         // Note: this check is necessary when word_size is not large enough to
         // cover all symbols of the field.
         // Following check is used for FFT over FNT where the single special
         // case symbol equals card - 1
-        if (data && *data == "@") {
+        if (data == 1) {
```

Comment:
Could we use a define/global constant instead of `1`.
Having a name would be more readable and explicit.

Reply:
it's done. Thanks